specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -4143,6 +4181,24 @@
       "type": "string",
       "in": "path",
       "x-ms-skip-url-encoding": true
+    },
+    "RecognitionModel": {
+      "name": "RecognitionModel",
+      "description": "Recognition model name. Recognition model is used when the face features are extracted, so a recognition model version could be provided when performing a Detection. If the user does not provide recognition model, default is 'recognition_v01'.",
+      "required": false,
+      "type": "string",
+      "in": "query",
+      "x-ms-parameter-location": "method",   
+      "default": "recognition_v01",
+      "x-nullable": false,
+      "x-ms-enum": {
+        "name": "RecognitionModel",
+        "modelAsString": false
+      },
+      "enum": [
+        "recognition_v01",
```

Comment:
I would suggest using simpler values (e.g. v01).

Reply:
Because we may also support multiple models for other face algorithms (e.g. detection models, even attribute models, etc.), we may prefer the prefix to specify the model type.

Reply:
Cool - thanks for the explanation.